src/lib/dragondeploy/index.ts

```diff
@@ -0,0 +1,239 @@
+import ArDB from 'ardb'
+import Arweave from 'arweave'
+import git, { WORKDIR } from 'isomorphic-git'
+import mime from 'mime'
+import type { Dispatch, SetStateAction } from 'react'
+import { InjectedArweaveSigner } from 'warp-contracts-plugin-signature'
+
+import { checkoutBranch } from '@/lib/git/branch'
+import { fsWithName } from '@/lib/git/helpers/fsWithName'
+import { Repo } from '@/types/repository'
+
+export const APP_NAME = 'Dragon-Deploy'
+export const APP_VERSION = '0.3.0'
+export const MANIFEST_CONTENT_TYPE = 'application/x.arweave-manifest+json'
```

Comment:
@pawanpaudel93 can we also add a tag to trace that the deployment came from Protocol.Land and also repo name repo id branch name author(address) who deployed this?